Add DockerDiscordControl template

DockerDiscordControl opened the pull request:
## DockerDiscordControl Template Submission

Adding comprehensive Docker container management through Discord bot with integrated web interface.

### Key Features:
- Discord slash commands for container control (start, stop, restart)
- Web interface with real-time monitoring and configuration
- Task scheduler for automated container actions (daily, weekly, monthly)
- Channel-based permissions and security framework
- Alpine Linux based - lightweight and secure
- Support for Buy Me A Coffee and PayPal donations

### Technical Details:
- Docker Hub: dockerdiscordcontrol/dockerdiscordcontrol
- Homepage: https://ddc.bot
- Web UI Port: 8374
- Categories: Tools, HomeAutomation
- Base Image: Alpine Linux (enhanced security, minimal resource usage)

### Use Case:
Perfect for Unraid servers and Discord communities managing Docker containers remotely. Combines the convenience of Discord bot commands with a comprehensive web interface for advanced configuration and monitoring.

### Prerequisites:
Users must create a Discord bot token before first use. Complete setup guide available at https://ddc.bot

---

**Contact:** max@ddc.bot  
Project maintainer available for template discussions and support.

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>DockerDiscordControl</Name>
+  <Repository>dockerdiscordcontrol/dockerdiscordcontrol</Repository>
```
Please specify an explicit default tag, even if it is the implicit `latest` tag.

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+  <Repository>dockerdiscordcontrol/dockerdiscordcontrol</Repository>
+  <Registry>https://hub.docker.com/r/dockerdiscordcontrol/dockerdiscordcontrol</Registry>
+  <Network>bridge</Network>
+  <MyIP/>
```
```suggestion
```

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+  <Overview>Control your Docker containers directly from Discord! Revolutionary performance optimizations with web UI and task scheduling. Alpine Linux based for enhanced security and minimal resource usage. Features Discord bot commands, automated scheduling, and real-time monitoring perfect for Unraid servers.</Overview>
+  <Category>Tools: HomeAutomation:</Category>
+  <WebUI>http://[IP]:[PORT:8374]/</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/DockerDiscordControl/DockerDiscordControl/main/unraid-template.xml</TemplateURL>
```
Template URL should point to this repo.

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/commands.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/control1.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/task1.png</Screenshot>  <ExtraParams>--restart=unless-stopped</ExtraParams>
+  <PostArgs/>
```
```suggestion
```

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/control1.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/task1.png</Screenshot>  <ExtraParams>--restart=unless-stopped</ExtraParams>
+  <PostArgs/>
+  <CPUset/>
```
```suggestion
```

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/task1.png</Screenshot>  <ExtraParams>--restart=unless-stopped</ExtraParams>
+  <PostArgs/>
+  <CPUset/>
+  <DateInstalled>1734711600</DateInstalled>
```
```suggestion
```

nwithan8 commented on templates/dockerdiscordcontrol.xml:
```diff
@@ -0,0 +1,94 @@
+      </Port>
+    </Publish>
+  </Networking>
+  <Data>
```
Recommend turning all these Data elements into Config elements

DockerDiscordControl commented:
**All review feedback implemented**

Hi @nwithan8,

Thank you for the thorough review! I've addressed all your feedback points:

**Implemented Changes:**
- Added explicit `<Default>latest</Default>` tag
- Removed unnecessary `<MyIP/>` tag  
- Fixed TemplateURL to point to this unRAID-CA-templates repo
- Removed empty `<PostArgs/>` tag
- Removed empty `<CPUset/>` tag
- Removed `<DateInstalled>` tag
- Converted all `<Data>` elements to `<Config>` elements for better unRAID integration

**Technical Status:**
- XML validation passes (`xmllint --noout`)
- All commits pushed with detailed messages
- Template follows unRAID Community Apps best practices

The template is now ready for approval and should integrate seamlessly with unRAID Community Applications.
